### PR TITLE
Token verify scope

### PR DESCRIPTION
**Pull request by iamigo**
Speed up the verify user token query

### CODE REVIEW

File: utils/jwtUtil.js [hunk 1]
```diff
@@ -254,13 +218,17 @@ function verifyUserToken(payload, req, cb) {
     // No need to check the token record if this is the default UI token.
     if (p.username === p.tokenname) return cb();
 
-    return Token.findOne({
-      where: {
-        name: p.tokenname,
-        createdBy: req.user.id,
-      },
+    return Token.scope({ method: ['notRevoked', p.tokenname, req.user.id] })
```
Comment (pallavi2209) [inline utils/jwtUtil.js]: Update or add test with updated error info?

**Comment by iamigo** [comment]
@pallavi2209 added test

File: tests/api/v1/userTokens/revokeRestore.js [hunk 2]
```diff
@@ -112,6 +116,30 @@ describe('tests/api/v1/userTokens/revokeRestore.js, ' +
...
+
+      expect(res.body).to.have.property('name', tname);
+      expect(res.body.isRevoked > '0').to.be.true;
+      api.get(path)
+      .set('Authorization', voldemortToken)
+      .send({})
+      .expect(constants.httpStatus.FORBIDDEN)
+      .end((err2, res2) => {
+        expect(res2.body.errors[0])
+          .to.have.property('description', 'Authentication Failed');
```
Comment (pallavi2209) [inline tests/api/v1/userTokens/revokeRestore.js]: Looks like you are not updating the description when throwing the error - but we have the description here. Is this error coming from somewhere else?

Comment (iamigo) [reply tests/api/v1/userTokens/revokeRestore.js]: Yeah I realized that verifyToken was swallowing that explanation and replacing it with generic "Authentication Failed" anyway, so I just got rid of the custom description there.

**Pull request merged**